tests/lib/rules/no-empty.js

```diff
@@ -44,36 +44,173 @@ ruleTester.run("no-empty", rule, {
+        {
+            code: "try {} catch (ex) {throw ex}",
+            errors: [{
+                messageId: "unexpected",
+                data: { type: "block" },
+                type: "BlockStatement",
+                suggestions: [{
+                    messageId: "suggestComment",
+                    output: "try { /* empty */ } catch (ex) {throw ex}"
+                }]
+            }]
+        },
+        {
+            code: "try { foo() } catch (ex) {}",
+            errors: [{
+                messageId: "unexpected",
+                data: { type: "block" },
+                type: "BlockStatement",
+                suggestions: [{
+                    messageId: "suggestComment",
+                    output: "try { foo() } catch (ex) { /* empty */ }"
+                }]
+            }]
+        },
+        {
+            code: "try { foo() } catch (ex) {throw ex} finally {}",
+            errors: [{
+                messageId: "unexpected",
+                data: { type: "block" },
+                type: "BlockStatement",
+                suggestions: [{
+                    messageId: "suggestComment",
+                    output: "try { foo() } catch (ex) {throw ex} finally { /* empty */ }"
+                }]
+            }]
+        },
+        {
+            code: "if (foo) {}",
+            errors: [{
+                messageId: "unexpected",
+                data: { type: "block" },
+                type: "BlockStatement",
+                suggestions: [{
+                    messageId: "suggestComment",
+                    output: "if (foo) { /* empty */ }"
+                }]
+            }]
+        },
+        {
+            code: "while (foo) {}",
+            errors: [{
+                messageId: "unexpected",
+                data: { type: "block" },
+                type: "BlockStatement",
+                suggestions: [{
+                    messageId: "suggestComment",
+                    output: "while (foo) { /* empty */ }"
+                }]
+            }]
+        },
+        {
+            code: "for (;foo;) {}",
+            errors: [{
+                messageId: "unexpected",
+                data: { type: "block" },
+                type: "BlockStatement",
+                suggestions: [{
+                    messageId: "suggestComment",
+                    output: "for (;foo;) { /* empty */ }"
+                }]
+            }]
+        },
+        {
+            code: "switch(foo) {}",
+            errors: [{
+                messageId: "unexpected",
+                data: { type: "switch" },
+                type: "SwitchStatement"
+            }]
+        },
+        {
+            code: "switch (foo) { /* empty */ }",
+            errors: [{
+                messageId: "unexpected",
+                data: { type: "switch" },
+                type: "SwitchStatement"
+            }]
+        },
```

Comment:
Can you also add `suggestions: null` in errors for switch statements, to assert that the rule in that case does not provide suggestions.

Reply:
Added 👍🏻